dev-preview/README.md

```diff
@@ -199,3 +200,172 @@ The OpenShift Cluster API Operator deploys the main components (CAPI & CAPA depl
 ### Search-v2 - Odyssey
 
   * Search-v2 brings a re-architected backbone facilitating greater scale and resiliance within the service.
+
+## Global Hub Integration with Management Fabric
+
+Management fabric is designed to establish a common hosted integration fabric and control plane which can be plugged into each Red Hat management product or service, as well as consumed by our ISV partners.
+
```

Comment:
A management service creates, modifies or deletes a managed resource in its local inventory. To aid in the integration of the Red Hat Hybrid Cloud Management solution, they will report all of these operations to the management fabric via the Asset Inventory APIs. With the Asset Inventory APIs, these services are referenced as Reporters.

What about this:

To aid the integration of the Red Hat Hybrid Cloud Management solutions, Management Fabric was created. A management solution like ACM creates, modifies or deletes a managed resource in its local inventory. Also, all these operations will be reported to the management fabric via the Asset Inventory APIs. Other Red Hat Hybrid Cloud Management solutions will follow. Customers and ISV partners will be able to plug into Management Fabric to subscribe to these change events.